CHANGELOG.md

```diff
@@ -7,21 +7,44 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
+
+- Replaced `HTTPoison` library with `Tesla`.
+
+### Removed
+
+- Removed `Agent` strategy in favor of configuration. See `t:Segment.options/0` for configuration
+  instructions.
+
+### Added
```

Comment:
Please update README with some usage information about what was added, at least linking the modules/functions documentation.